gallery_dl/extractor/boosty.py

```diff
@@ -367,3 +468,32 @@ def _pagination_users(self, endpoint, params):
             if offset > data["total"]:
                 return
             params["offset"] = offset
+
+    def dialog(self, dialog_id):
+        endpoint = "/v1/dialog/{}".format(dialog_id)
+        return self._call(endpoint)
+
+    def dialog_messages(self, dialog_id, limit=300, offset=None):
+        endpoint = "/v1/dialog/{}/message/".format(dialog_id)
+        params = {
+            "limit": str(limit),
+            "reverse": "true",
+            "offset": str(offset) if offset else None,
+        }
+        return self._pagination_dialog(endpoint, params)
+
+    def _pagination_dialog(self, endpoint, params):
+        all_messages = []
```

Comment:
What's this for? You are collecting all `message` objects to *not* actually use them?